apps/frontend/components/pages/Form/FormValidationContainer.tsx

```diff
@@ -20,6 +26,35 @@ export const FormValidationContainer = (): JSX.Element => {
       ) : (
         <FormValidation validationForm={form} />
       )}
+      <Box>
+        <Box>{JSON.stringify(form?._validation_status)}</Box>
```

Comment:
at this point form should always be defined, is this "?" for typescript?

Reply:
It is outside of the ternary here